### DISCUSSION TITLE
Add device name variable for scripting

### DISCUSSION THREAD

**Discussion by chriscn** [arendst/Tasmota#10433] [Ideas]
It would be nice to be able to access the device name using the [special variables](https://tasmota.github.io/docs/Scripting-Language/#special-variables) section. I understand there is `frnm` for the Friendly Name. However perhaps `dvnm` for device name.

**Comment by gemu2015** [comment]
ok, is implemented 
it will be included in next pr

**Comment by chriscn** [comment]
Awesome thank you. I did have a look at the code but not very skilled with C.
